.azure-pipelines.yml

```diff
@@ -25,6 +25,14 @@ jobs:
         parameters:
           crate_path: '.'
 
+      - template: ci/build.yml
+        parameters:
+          crate_path: 'druid-shell'
```

Comment:
I'm not sue this is necessary? I think that because druid builds druid-shell, the warnings will be propagated.

Reply:
I suspected that as well, but it didn't seem to be the case when I was testing it locally.

```
C:\Users\hilma\git\druid
λ cargo rustc -- -D warnings
   Compiling druid-shell v0.3.0 (C:\Users\hilma\git\druid\druid-shell)
warning: unused variable: `warning`
  --> druid-shell\src\window.rs:37:13
   |
37 |         let warning = "warning";
   |             ^^^^^^^ help: consider prefixing with an underscore: `_warning`
   |
   = note: #[warn(unused_variables)] on by default

   Compiling druid v0.3.0 (C:\Users\hilma\git\druid)
    Finished dev [unoptimized + debuginfo] target(s) in 8.89s
```

Reply:
Okay cool, thanks. It’s possible there’s another compiler flag for this but I’m satisfied as-is!

Reply:
Yeah, there is an environment variable for denying warnings, but this fails on warnings on any dependency, which I'm not sure is desired behavior?

Reply:
yea I could see that causing headaches.